components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -166,6 +166,7 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
                 type: "regular",
                 creationTime: new Date().toISOString(),
                 contextURL: normalizedContextURL,
+                projectId: context.prebuiltWorkspace.projectId,
```

Comment:
Really? So there is no sharing of prebuilds between teams? Is that enforced somewhere?

Reply:
The current logic is that we have a single project per repo in Gitpod and workspaces are assigned to a project no matter you are a member or not. Happy to discuss 📞  😄

Reply:
@geropl this association does not reflect the runtime behavior. finding and using a prebuild to start a workspace from may follow other rules.

Reply:
Discussed offline and I now understand that:
 - project is owned by exactly one team
 - project is a GH repo
 - thus there is a 1-1 relation between workspace and project (atm, at least.